gobblin-data-management/src/main/java/gobblin/data/management/conversion/hive/AvroSchemaManager.java

```diff
@@ -110,7 +113,22 @@ public void cleanupTempSchemas() throws IOException {
   }
 
   public static Schema getSchemaFromUrl(Path schemaUrl, FileSystem fs) throws IOException {
```

Comment:
Correction: Changing this to ```URI``` will need wider refactor as ```SerializableHivePartition``` and ```SerializableHiveTable``` use ```Path``` as schemaUrl. 
I suggest moving this logic to read from http/https into ```getOrGenerateSchemaFile```. So we would be creating a temp schema file on HDFS by reading the schema from registry. This way we are consistent with other cases and we guarantee schema used in source is same as in extractor.